tests/TestData/StripeWebhook/ApiResponse/pm.json

```diff
@@ -24,7 +24,7 @@
     "country": "US",
     "exp_month": 8,
     "exp_year": 2024,
-    "fingerprint": "Xt5EWLLDS7FJjR1c",
+    "fingerprint": "TEST_CARD_FINGERPRINT",
```

Comment:
Good call!

Reply:
I was confused for a minute because `Xt5EWLLDS7FJjR1c` was only mentioned once in the test file, I thought maybe the code under test was computing it somehow.